src/ServicePulse.Host/app/modules/monitoring/js/services/services.messageTypeParser.js

```diff
@@ -30,7 +31,11 @@
                     `${item.typeName} |${item.assemblyName}-${item.assemblyVersion}` + (item.culture ? ` |${item.culture}` : '') + (item.publicKeyToken ? ` |${item.publicKeyToken}` : ''),
                     '');
             } else {
-                var tooltip = `${messageType.typeName} | ${messageType.assemblyName}-${messageType.assemblyVersion}`;
+                //Get the name without the namespace and remove nested types
+                var nestedCleanup = messageType.typeName.replace('+', '.');
```

Comment:
should we really remove nested type information? That's not necessarily part of the namespace but of the actual type name, isn't it?